interactive_engine/executor/ir/clients/rust/client/src/physical_builder.rs

```diff
@@ -209,6 +209,15 @@ impl Plan {
         self
     }
 
+    // Notice that this is used to set the meta_data of the **Last Appended OP**
+    pub fn with_meta_data(&mut self, meta_data: Option<Vec<pb::physical_opr::MetaData>>) {
```

Comment:
Why `Vec` requires `Option`? Empty `Vec` stands for `None`?

Reply:
Done.